OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -427,11 +427,7 @@ void SelectServer(GameServer server)
 					.ToDictionary(c => c.SpawnPoint, c => new SpawnOccupant(c, server.Mod != modData.Manifest.Id));
 
 				mapPreview.SpawnOccupants = () => occupants;
-
-				// TODO: Show disabled spawns
-				// This requires further changes to MasterServerPinger and the master server backend
-				var noSpawns = new List<int>();
-				mapPreview.DisabledSpawnPoints = () => noSpawns;
+				mapPreview.DisabledSpawnPoints = () => server.DisabledSpawnPoints ?? new List<int>();
```

Comment:
This will create a new list every time the UI is rendered if `server.DisabledSpawnPoints` is null!

We could either make sure that `server.DisabledSpawnPoints` like I suggested above for replays, otherwise pull out a `var disabledSpawns = server.DisabledSpawnPoints ?? new List<int>();` and return that like we do for `occupants`.